tests/CMakeLists.txt

```diff
@@ -48,6 +48,8 @@ endif()
 
 # Debug mode for tests
 set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "" FORCE)
+# Make sure ctest gives the output when tests fail
+list(APPEND CMAKE_CTEST_ARGUMENTS "--output-on-failure")
```

Comment:
Nice!! Hiding output is one of the worst things I know (at least in the area of test frameworks). It makes it impossible to debug.

Reply:
this only works for newer cmake though ...  CI runs with:
make CTEST_OUTPUT_ON_FAILURE=1 test 
to make it work for older cmakes..